.tekton/pipelinerun.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: spectral-lint
+  namespace: tekton-tasks
+spec:
+  pipelineRef:
+    name: openapi-spec
+  workspaces:
+  - name: shared-data
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+  params:
+  - name: repo-url
+    value: https://github.com/janus-api-idp/api-spec-prototype.git
+  - name: gitRevision
+    value: add-spectral-pipeline
+  - name: image-reference
+    value: quay.io/maximilianb/spectral:latest
```

Comment:
Why not use the default openshift image container ?

Reply:
Because the goal was to test things quickly. It can be this or the internal registry from a testing standpoint it's fine. When we will have a full integration with platform components it will be the case anyway